backend/api/tests/test_models.py

```
@@ -0,0 +1,222 @@
+import pytest
+from django.core.exceptions import ValidationError
```

Ruff (0.8.2)

`django.core.exceptions.ValidationError` imported but unused

Remove unused import: `django.core.exceptions.ValidationError`

(F401)